apps/src/code-studio/pd/application_dashboard/detail_view_contents.jsx

```diff
@@ -599,13 +599,14 @@ export class DetailViewContents extends React.Component {
   };
 
   renderStatusSelect = () => {
-    // Nobody is able to set an application status to incomplete from detail view
+    // Nobody is able to set an application status to incomplete from detail view,
```

Comment:
I'm confused by this comment

Reply:
Here's the screenshot: the red arrow is where RPs can change the status of an application––"incomplete" should not be one of the options.
<img width="1277" alt="Screen Shot 2022-02-28 at 9 35 15 AM" src="https://user-images.githubusercontent.com/9142121/156001861-e15f875d-8da0-4a3e-a26f-efd0e88d4610.png">

BUT that dropdown is also the easiest way to see what the current status is based on the default value:
<img width="1264" alt="Screen Shot 2022-02-28 at 9 39 43 AM" src="https://user-images.githubusercontent.com/9142121/156002141-db496b2e-006f-4c7c-a5c3-526bf3202371.png">
So if the current status is incomplete, they should be able to see that in the dropdown.

If you have ways to make this clearer, I'm all ears

Reply:
Thanks for explaining. I think my confusion may have been because I'm not familiar with this area. One thought is that maybe phrase it in the positive might be helpful "Only show Incomplete in the dropdown if application is incomplete."

Reply:
Good idea––thanks!